libs/core/src/lib/utils/pipes/safe.pipe.ts

```diff
@@ -0,0 +1,26 @@
+import { Pipe, PipeTransform } from '@angular/core';
+import { DomSanitizer, SafeHtml, SafeStyle, SafeScript, SafeUrl, SafeResourceUrl } from '@angular/platform-browser';
+
+@Pipe({
+    name: 'safe'
+})
+
+export class SafePipe implements PipeTransform {
```

Comment:
Hm, if I understand the `bypassSecurityTrust*` methods well, this is used to prevent angular from checking for potential dangers in passed link etc. Is this what we really want to do here? If yes, then why?

Reply:
I implemented this pipe for author link, `authorLink | safe : 'url'`
And decided it would be helpful to add other for future implementations. If it's unnecessary, I will remove others.